.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: klaytn-safe-react deploy workflow
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_ARN_DEV }}
+          role-session-name: SessionForKlaytnActions
+          aws-region: ${{ secrets.AWS_REGION_DEV }}
+      - name: Installing Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+      - name: Installing dependencies
+        run: yarn install
+      - name: Change homepage and get .env
+        env:
+          PARAMETER_STORE_DEV: ${{ secrets.PARAMETER_STORE_DEV }}
+        run: |
+          sed -i -e "s+\/app+\/+g" package.json

```

Comment:
Please show me the example of what transition will be held for this regular expression.

Reply:
This code is very tricky. Why do we need this line?